src/components/dashboardNavbar.js

```diff
@@ -0,0 +1,126 @@
+import Grid from "@material-ui/core/Grid";
+import React from "react";
+import PiggyLogo from "../assets/logo/piggy-logo.svg";
+import InputBase from '@material-ui/core/InputBase';
+import SearchIcon from '@material-ui/icons/Search';
+import NotificationsNoneOutlinedIcon from '@material-ui/icons/NotificationsNoneOutlined';
+import Avatar from '@material-ui/core/Avatar';
+import {
+  Box,
+  fade,
+  makeStyles,
+} from "@material-ui/core"
+
+const useStyles = makeStyles(theme => ({
+    logoWrap: {
+      maxHeight: 50,
+      width: 120,
+      [theme.breakpoints.up("sm")]: {
+        width: 100,
+      },
+      [theme.breakpoints.up("md")]: {
+        width: 120,
+      },
+      },
+    search: {
+      position: 'relative',
+      borderRadius: theme.shape.borderRadius,
+      backgroundColor: '#F5F6F9',
+      '&:hover': {
+        backgroundColor: fade('#F5F6F9', 0.25),
+      },
+      marginRight: theme.spacing(2),
+      marginLeft: 0,
+      width: '100%',
+      [theme.breakpoints.up('sm')]: {
+        marginLeft: theme.spacing(3),
+        width: 'auto',
+      },
+    },
+    searchIcon: {
+      padding: theme.spacing(0, 2),
+      height: '100%',
+      position: 'absolute',
+      pointerEvents: 'none',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      color: '#8E8E93',
+    },
+    notificationsIcon: {
+      padding: theme.spacing(0, 2),
+      height: '100%',
+      position: 'absolute',
+      pointerEvents: 'none',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    inputRoot: {
+      color: 'inherit',
+    },
+    inputInput: {
+      padding: theme.spacing(1, 1, 1, 0),
+      // vertical padding + font size from searchIcon
+      paddingLeft: `calc(1em + ${theme.spacing(4)}px)`,
+      transition: theme.transitions.create('width'),
+      width: '100%',
+      [theme.breakpoints.up('md')]: {
+        width: '20ch',
+      },
+      notificationsBorders: {
+        border: '1',
+        borderColor: '#D8D8D8',
+        borderLeft: '1',
+        borderRight: '1',
+      }
+    },
+}))
+
+export default function DashboardNavbar() {
+  const classes = useStyles()
+  return (
+    <div>
```

Comment:
Look into how the app bar with a primary search field works. You have some snippets from the example but something is clearly off here. https://material-ui.com/components/app-bar/#app-bar-with-a-primary-search-field